pkgs/applications/misc/kjv/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv, fetchFromGitHub, fetchpatch }:
+
+stdenv.mkDerivation rec {
+  pname = "kjv";
+  version = "unstable-2018-12-25";
+
+  src = fetchFromGitHub {
+    owner = "bontibon";
+    repo = pname;
+    rev = "fda81a610e4be0e7c5cf242de655868762dda1d4";
+    sha256 = "1favfcjvd3pzz1ywwv3pbbxdg7v37s8vplgsz8ag016xqf5ykqqf";
+  };
+
+  add-apocrypha = fetchpatch {
```

Comment:
You can inline this in `patches` like
```
patches = [
  (fetchpatch {
    url = ...
    sha256 = ...
  })
];
```

Reply:
I tend to favour explicit bindings (in the spirit, if not the letter, of [A-normal form](https://en.wikipedia.org/wiki/A-normal_form)).